docs/Intro/dive/onchain.md

```diff
@@ -2,22 +2,19 @@
 
-    - **ValidatorRegistry** plays a crucial role within the **Chronicle Protocol** by maintaining a comprehensive list of all Validators that are recognized as valid participants in the network/protocol. Validators, in this context, are trusted entities within the network that collaborate through a *peer-to-peer* (p2p) network to supply new oracle data.
+    - **ConfigRegistry**: This is a Go-based Oracle client that stores the configurations required for validators.
 
-    - An on-chain enforced consensus mechanism ensures that data provided by these Validators is validated by a specific number, known as the `bar` of Validators. As a singleton contract deployed on the Ethereum blockchain, the **ValidatorRegistry** serves as a centralized source of truth regarding the identities of valid Validators within the Chronicle Protocol. Validators regularly query this registry to determine their status within the network. Additionally, the registry provides functionalities such as filtering p2p connections based on the identities of Validators and translating validator addresses to/from their corresponding 1-byte identifier.
-    By leveraging the **ValidatorRegistry**, Chronicle Protocol ensures that only authenticated and authorized Validators participate in the network activities, thereby enhancing the security and integrity of the oracle data provided by the protocol.
 
+    - **ValidatorRegistry** plays a crucial role within the **Chronicle Protocol** by maintaining a comprehensive list of all Validators that are recognized as valid participants in the network/protocol. Validators, in this context, are trusted entities within the network that collaborate through a *peer-to-peer* (p2p) network to supply new oracle data.
 
+    - An onchain enforced consensus mechanism ensures that data provided by these Validators is validated by a specific number, known as the `bar` of Validators. As a singleton contract deployed on the Ethereum blockchain, the **ValidatorRegistry** serves as a centralized source of truth regarding the identities of valid Validators within the Chronicle Protocol. Validators regularly query this registry to determine their status within the network. Additionally, the registry provides functionalities such as filtering p2p connections based on the identities of Validators and translating validator addresses to/from their corresponding 1-byte identifier.
```

Comment:
I don;t think it's a singleton... @pmerkleplant ?

Reply:
It is a singleton, the contract exists exactly once only on Ethereum.